File: pset6/mario/mario.py
from cs50 import get_int

# Declaring variables
space = ' '
hsh = '#'
n = 1

# Getting height
height = get_int("Height: ")

# Check if height is right
while height <= 0 or height >= 9:
    height = get_int("Height: ")

# constructing the pyramid
for i in range(height):
    print(space * (height - (i + 1)), end='')
    print(hsh * (i + 1), end='')
    print()

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Declaring variables
    int height = 0;
    char space = ' ', hash = '#';
    
    while (height <= 0 || height >= 9)
    {
        //Take the height
        height = get_int("Height: ");
    }
    //Conditional to the height number


    //Loop that makes the pyramid
    
    for (int n = 1; n <= height; n++)
    {
        for (int i = 1; i <= height - n ; i++)
        {
            printf("%c", space);
        }
    
        for (int j = 1; j <= n; j++)
        {
            printf("%c", hash);
        }
        printf("\n");
    }
}